performance-metrics/src/performance_tests.rs

```diff
@@ -41,6 +40,24 @@ impl From<InfraError> for Error {
     }
 }
 
+const BLK_IO_TEST_IMG: &str = "/home/ch-blk-io-test.img";
```

Comment:
Perhaps you could use `/var/tmp` here too.

Reply:
Changes adopted.